src/middleware-wrapper.js

```diff
@@ -25,7 +25,13 @@ const middlewareWrapper = config => {
 
     if (req.path === config.path) {
       if (config.iframe) {
-        res.remove(' X-Frame-Options');
+        if(res.removeHeader){
```

Comment:
This PR has linter errors. You need to include a space after your `if` statement and another space after your parenthesis.